html/includes/graphs/application/ntp-server_freq.inc.php

```diff
@@ -4,8 +4,8 @@
 
 $scale_min       = 0;
 $ds              = 'frequency';
-$colour_area     = 'F6F6F6';
-$colour_line     = 'B3D0DB';
+$colour_area     = $config['graph_colours']['pinks']['0'].'33';
+$colour_line     = $config['graph_colours']['pinks']['1'];
```

Comment:
If you are setting the alpha to 33 for the area, shouldn't these both be the same index?